bundles/org.openhab.binding.caddx/README.md

```diff
@@ -0,0 +1,246 @@
+# Caddx Binding
+
+The Caddx binding is used for communicating with the Caddx alarm panels
+
+It provides connectivity to the alarm panel via a RS-232 serial connection to the Caddx interface or directly to the NX8E.
+
+
+## Supported Things
+
+This binding supports the following Thing types
+
+| Thing      | Thing Type | Description                                                            |
+|------------|------------|------------------------------------------------------------------------|
+| bridge     | Bridge     | The  RS-232 interface.                                                 |
+| panel      | Thing      | The basic representation of the alarm System.                          |
+| partition  | Thing      | Represents a controllable area within the alarm system.                |
+| zone       | Thing      | Represents a physical device such as a door, window, or motion sensor. |
+| keypad     | Thing      | Represents a keypad.                                                   |
```

Comment:
Can you mention, that keypad is not implemented, yet?

Reply:
Since you mentioned you can't see my comments anymore, I'm commenting on this, so you might see this again.

Reply:
Done